cheatsheet.md

```diff
@@ -74,25 +46,42 @@ a `rendersubtreeactivation` signal is fired on the element and the `rendersubtre
 property is set to `""`. Note that `skip-activation` implies
 `skip-viewport-activation` when disabling a particular algorithm.
 
-* find-in-page: when find-in-page finds an element and makes it an active match
-  (current match), the locked ancestors of the element are activated. (disabled
-  by `skip-activation`)
-* viewport intersection: when a locked element enters the viewport, it is
-  activated (disabled by `skip-viewport-activation`)
-* scrollIntoView(): when scrollIntoView() is called on an element, the locked
-  ancestors of the element are activated (disabled by `skip-activation`)
-* accessibility: screen readers and other accessibility features activate
-  elements (disabled by `skip-activation`)
-* anchor link navigation: when fragment links (ie url.html#elementid) navigates
-  to an element, its locked ancestors are activated (disabled by
-  `skip-activation`)
-* tab order navigation: when an element is focused by tab navigation (forward or
-  backward), the locked ancestors of the focused element are activated (disabled
-  by `skip-viewport-activation`)
-* selection: when an element is selected, it is activated so its children can be
-  selected as well (disabled by `skip-viewport-activation`)
-* focus(): when focus() is called on an element, its locked ancestors are
-  activated (disabled by `skip-activation`)
+### Viewport intersection
+* When a locked element enters the viewport or nearing the viewport (50% margin), it will be activated.
+* Can be disabled with `skip-viewport-activation` or `skip-activation`.
+
+### Selection
+* When text in locked subtree gets selected, all of the text's locked ancestors will be activated.
+* Can be disabled with `skip-viewport-activation` or `skip-activation`.
+ 
+### Sequential/tab-order focus navigation
+* When an element is focused by sequential focus navigation (forward or backward),
+the locked ancestors of the focused element will be activated
+* Can disabled with `skip-viewport-activation` or `skip-activation`.
+
+### Find-in-page
+* Find-in-page will find text even if they are in a locked subtree (but will skip locked subtrees with `skip-activation`), counting them in the total match.
```

Comment:
s/they are/it is/

I would maybe refrase as "... even if it is in a locked activatable subtree ...". With possibly a note before this whole block explaining the difference between locked activatable and locked non-activatable

Reply:
Rephrased a bit & added more explanation.